config/locales/models/criteria/fr.yml

```diff
@@ -30,6 +30,9 @@ fr:
         ta_visible: "Rendre visible aux assistants d'enseignement"
       rubric_criterion:
         description: "Description"
+        level_name: "Nom"
```

Comment:
This is a good start but in the wrong place. You need to create *new* files to represent the strings for a `Level` model.